### ISSUE TITLE
Test results are duplicated in output

### ISSUE THREAD

**Issue by michaelleeallen** [michaelleeallen/mocha-junit-reporter#2]
I'm not even sure if this is an issue anymore. Can anyone verify if this issue still exists?

**Comment by lazytesting** [comment]
I'm not able to reproduce this, I think it is fixed by https://github.com/michaelleeallen/mocha-junit-reporter/pull/3.

Output for a single test using version 1.3.1:
```xml
<?xml version="1.0" encoding="UTF-8"?>
<testsuites>
<testsuite name="Product Detail Page" tests="1" failures="0" timestamp="2015-05-08T07:00:36" time="0">
<testcase name="Product Detail Page should have the correct page title" time="16.148" className="should have the correct page title">
</testcase>
</testsuite>
</testsuites>
```

**Comment by michaelleeallen** [comment]
Fixed by #3
